cni/pkg/iptables/iptables.go

```diff
@@ -478,38 +568,24 @@ func (cfg *IptablesConfigurator) CreateHostRulesForHealthChecks(hostSNATIP, host
 	return nil
 }
 
-func (cfg *IptablesConfigurator) DeleteHostRules() {
+func (cfg *IptablesConfigurator) DeleteHostRules(hostSNATIP, hostSNATIPV6 netip.Addr) {
```

Comment:
Can be a followup, but wouldn't we want to reconcile this instead of delete + create?

Reply:
We actually then later say `found residues of old iptables rules/chains, reconciliation is needed   component=host`.. but then it always fails since Reconcile is hardcoded to `false`

Reply:
> Can be a followup, but wouldn't we want to reconcile this instead of delete + create?

Yep, he had this in originally. I agree we definitely want it as a followup. I asked to remove it, since hostrules don't impact dataplane traffic, so delete+recreate is safe.